### PR TITLE
Implement support for not writing to disk and fetching measurements f…

### PR DESCRIPTION
**Pull request by hellais**
…rom the API

### CODE REVIEW

**Comment by hellais** [comment]
TODO:
* ooni/probe-engine does not supporta raw_measurement API
* if the API is blocked, it will not work to download the measurement even if it was uploaded (using something like cloudfront)

File: internal/database/actions.go [hunk 1]
```diff
@@ -55,6 +56,25 @@ func GetMeasurementJSON(sess sqlbuilder.Database, measurementID int64) (map[stri
 		log.Errorf("failed to run query %s: %v", req.String(), err)
 		return nil, err
 	}
+	if measurement.IsUploaded {
+		// TODO this should be a function exposed by probe-engine
```
Comment (bassosimone) [inline internal/database/actions.go:60]:
```suggestion
		// TODO(bassosimone): this should be a function exposed by probe-engine
```

**Review by bassosimone — APPROVED** [review]
🐳

**Comment by bassosimone** [comment]
Thank you for working on this, @hellais! I am going to merge as soon as automatic checks are green!

**Comment by bassosimone** [comment]
This PR implements https://github.com/ooni/probe/issues/1313.

**Pull request merged**